localstack/services/dynamodbstreams/dynamodbstreams_api.py

```diff
@@ -139,7 +139,7 @@ def stream_name_from_stream_arn(stream_arn):
 
 def random_id(stream_arn, kinesis_shard_id):
     namespace = uuid.UUID(bytes=hashlib.sha1(to_bytes(stream_arn)).digest()[:16])
-    return uuid.uuid5(namespace, kinesis_shard_id).hex
+    return uuid.uuid5(namespace, kinesis_shard_id.encode('UTF-8')).hex
```

Comment:
Can we please use the `to_bytes` util function here (like in the line above):
```
return uuid.uuid5(namespace, to_bytes(kinesis_shard_id)).hex
```

Reply:
done